samples/snippets/quickstart_analyzeiampolicylongrunning_test.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import uuid
+
+from google.cloud import bigquery
+from google.cloud import storage
+
+import pytest
+
+import quickstart_analyzeiampolicylongrunning
+
+PROJECT = os.environ["GOOGLE_CLOUD_PROJECT"]
+BUCKET = "analysis-{}".format(uuid.uuid4().hex)
+DATASET = "analysis_{}".format(int(uuid.uuid4()))
+
+
+@pytest.fixture(scope="module")
+def storage_client():
+    yield storage.Client()
+
+
+@pytest.fixture(scope="module")
+def bigquery_client():
+    yield bigquery.Client()
+
+
+@pytest.fixture(scope="module")
+def analysis_bucket(storage_client):
+    storage_client.create_bucket(BUCKET)
+
+    yield BUCKET
+
+
+@pytest.fixture(scope="module")
+def dataset(bigquery_client):
+    dataset_id = "{}.{}".format(PROJECT, DATASET)
+    dataset = bigquery.Dataset(dataset_id)
+    dataset.location = "US"
+    dataset = bigquery_client.create_dataset(dataset)
+
+    yield DATASET
+
+    bigquery_client.delete_dataset(
+        dataset_id, delete_contents=True, not_found_ok=False)
+
+
+def test_analyze_iam_policy(analysis_bucket, dataset, capsys):
+    dump_file_path = "gs://{}/analysis-dump.txt".format(analysis_bucket)
+    quickstart_analyzeiampolicylongrunning.analyze_iam_policy_longrunning_gcs(PROJECT, dump_file_path)
```

Comment:
Is this a long running operation? Then is it possible to wait for the result somehow?

Reply:
Done. Wait until there's result.